File: ui/app/commons/model/RepoInfo.ts
export class RepoInfo {

    constructor(private repoUsername?: string,
                private repoPassword?: string,
                private repoUrl?: string){
    }

    getRepoUrl(): string {
        return this.repoUrl;
    }

    setRepoUrl(url: string): void {
        this.repoUrl = url;
    }

    getRepoUsername(): string {
        return this.repoUsername;
    }

    setRepoUsername(username: string): void {
        this.repoUsername = username;
    }

    getRepoPassword(): string {
        return this.repoPassword;
    }

    setRepoPassword(pass: string): void {
        this.repoPassword = pass;
    }
}

File: server/src/main/java/blamehub/shared/model/RepoInfo.java
package blamehub.shared.model;

/**
 * Created by dviyi01 on 8/19/2016.
 *
 */
public class RepoInfo {

    String repoUrl;
    String repoUsername;
    String repoPassword;

    public String getRepoUrl() {
        return repoUrl;
    }

    public void setRepoUrl(String repoUrl) {
        this.repoUrl = repoUrl;
    }

    public String getRepoUsername() {
        return repoUsername;
    }

    public void setRepoUsername(String repoUsername) {
        this.repoUsername = repoUsername;
    }

    public String getRepoPassword() {
        return repoPassword;
    }

    public void setRepoPassword(String repoPassword) {
        this.repoPassword = repoPassword;
    }

}
